### PR TITLE
dind: move to hyperkube and 'openshift start network'

### PR DESCRIPTION
**Pull request by dcbw**
Let's win half the battle and do the whole daemonset/image thing as the second part.

@openshift/sig-networking @squeed @danwinship @JacobTanenbaum

### CODE REVIEW

**Comment by imcsk8** [comment]
LGTM

File: images/dind/node/openshift-sdn-node.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  token=$(cat ${master_dir}/openshift-sdn.token)
+
+  # Take over network functions on the node
+  rm -Rf /etc/cni/net.d/80-openshift-network.conf
+
+  # use either the bootstrapped node kubeconfig or the static configuration
+  if [[ ! -f "${kube_config}" ]]; then
+    # use the static node config if it exists
+    # TODO: remove when static node configuration is no longer supported
+    for f in ${config_dir}/system*.kubeconfig; do
```
Comment (Miciah) [inline images/dind/node/openshift-sdn-node.sh]: I'd add a `shopt -s failglob` command here, or an additional check after the `for` loop to make sure `$kube_config` exists.  (I was surprised to find that `oc config --config=non-existent-file view --flatten` does not fail.)

File: images/dind/master/ovn-kubernetes-master.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@ function ovn-kubernetes-master() {
 
   local master_config="${config_dir}/master-config.yaml"
   cluster_cidr=$(python -c "import yaml; stream = file('${master_config}', 'r'); y = yaml.load(stream); print y['networkConfig']['clusterNetworks'][0]['cidr']")
-  apiserver=$(oc --config="${kube_config}" config view -o custom-columns=server:clusters[0].cluster.server | grep http)
+  apiserver=$(grep server ${kube_config} | cut -f 6 -d' ')
```
Comment (Miciah) [inline images/dind/master/ovn-kubernetes-master.sh]: Adding `-m1`, adding a colon to the pattern, and using `awk` would make the command more robust:
```bash
apiserver=$(grep -m1 server: /etc/origin/master/admin.kubeconfig | awk '{print $2}')
```

File: images/dind/node/openshift-sdn-node.sh (hunk 1 above)
Comment (dcbw) [reply images/dind/node/openshift-sdn-node.sh]: @Miciah yeah, I was surprised too.  Anyway, added a post-for check.

File: images/dind/master/ovn-kubernetes-master.sh (hunk 2 above)
Comment (dcbw) [reply images/dind/master/ovn-kubernetes-master.sh]: @Miciah fixed, thanks.

**Comment by dcbw** [comment]
@Miciah @danwinship updated; PTAL thanks!

**Comment by dcbw** [comment]
@Miciah @danwinship repushed to fix merge conflict, another LGTM would be great thanks!

**Pull request merged**